README.md

```diff
@@ -431,6 +451,7 @@ Here are some examples of making various layer level edits.
 ## How to use the APIs
 
 The API's are documented at https://adobedocs.github.io/photoshop-api-docs-pre-release/
+Sample examples are using the Input File [Example.psd](https://github.com/AdobeDocs/photoshop-api-docs-pre-release/blob/master/sample_files/Example.psd). You can download the file and store in your storage.
```

Comment:
"The code snippets are using one of our [sample psd files](https://github.com/AdobeDocs/photoshop-api-docs-pre-release/tree/master/sample_files). Please feel free to download and use it for testing. Just remember you will need to have this file stored in one of the accepted external solutions. For more information on storage please refer to the [File Storage](https://github.com/AdobeDocs/photoshop-api-docs-pre-release#input-and-output-file-storage) section.